components/Cards/src/MDCCardCollectionCell.h

```diff
@@ -0,0 +1,158 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "MDCCard.h"
+
+typedef NS_ENUM(NSInteger, MDCCardCellState) {
+  /** The visual state when the cell is in its normal state. */
+  MDCCardCellStateNormal = 0,
+
+  /** The visual state when the cell is in its highlighted state. */
+  MDCCardCellStateHighlighted,
+  
+  /** The visual state when the cell has been selected. */
+  MDCCardCellStateSelected
+};
+
+@interface MDCCardCollectionCell : UICollectionViewCell
+
+/**
+ When selecting is set to YES, a tap on a cell will trigger a visual change between selected
+ and unselected. When it is set to NO, a tap will trigger a normal tap (rather than trigger
+ different visual selection states on the card).
+ Default is set to NO.
+ */
+@property(nonatomic, assign, getter=isSelecting) BOOL selecting;
```

Comment:
I liked selectable better

Reply:
Naming is hard, but selectable / isSelectable seems a bit more accurate.

Reply:
yep, will rename, forgot the exact name that we decided